src/core/include/mp-units/math.h

```diff
@@ -144,10 +145,24 @@ template<auto R, typename Rep>
 }
 
 /**
- * @brief Determines if a number is infinite.
+ * @brief Determines if a quantity point is finite.
  *
- * @param a: Number to analyze.
- * @return bool: Whether the number is infinite or not.
+ * @param a: Quantity point to analyze.
+ * @return bool: Whether the quantity point is finite or not.
+ */
+template<auto R, auto PO, typename Rep>
+  requires requires(Rep v) { isfinite(v); } || requires(Rep v) { std::isfinite(v); }
```

Comment:
Maybe something like this?

```suggestion
  requires requires(Quantity<R, Rep> q) { isfinite(q); }
```